src/video/renderers/opengl3/gl3_renderer.c

```diff
@@ -223,8 +228,11 @@ static inline void video_set_blend_mode(gl3_context *ctx, object_array_blend_mod
         glColorMask(GL_TRUE, GL_TRUE, GL_TRUE, GL_TRUE);
     } else if(request_mode == MODE_ADD) {
         glColorMask(GL_FALSE, GL_FALSE, GL_FALSE, GL_TRUE);
-    } else {
+    } else if(request_mode == MODE_REMAP) {
         glColorMask(GL_FALSE, GL_TRUE, GL_TRUE, GL_TRUE);
+    } else if(request_mode == MODE_LOOPBACK) {
+        glColorMask(GL_TRUE, GL_TRUE, GL_TRUE, GL_TRUE);
+        texture_update_from_framebuffer(TEX_UNIT_LOOPBACK, ctx->tex_loopback, 0, 0, 0, 0, NATIVE_W, NATIVE_H);
```

Comment:
video_set_blend_mode doesn't seem like a correct plate to fiddle with texture copies